goblet/app.py

```diff
@@ -76,17 +77,23 @@ def local_func(request):
 
     def deploy(
         self,
-        skip_resources=False,
+        skip_handlers=False,
         skip_backend=False,
         skip_infra=False,
         force=False,
         write_config=False,
         stage=None,
+        handlers=None,
+        infras=None,
     ):
         g.config.update_g_config(values={"labels": self.labels})
         source = None
         backend = self.backend
-        if not skip_infra:
+
+        if infras:
+            log.info("deploying infrastructure")
+            self.deploy_infrastructure(infras)
```

Comment:
can we pass in infras either way with `skip_infra` since `deploy_infrastructure` can handle None already?

Reply:
yes that would work, will get it updated